recipes/unicodecsv/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "unicodecsv" %}
+{% set version = "0.14.1" %}
+{% set sha256 = "018c08037d48649a0412063ff4eda26eaa81eff1546dbffa51fa5293276ff7fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+package:
+  name: unicodecsv
+  version: "0.14.1"
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - unicodecsv
+
+about:
+  home: https://github.com/jdunck/python-unicodecsv
+  license: BSD
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Even though the package looks like it hasn't been updated in a while, could you submit an issue/PR mentioning the missing license file in the source distribution and put the url for the issue/PR in a comment in the recipe as a reminder to see if the situation has changed down the road.